adds a configuration setting to disable app metrics for specific apps (#5493)
nautobot/nautobot: nautobot/docs/user-guide/administration/guides/prometheus-metrics.md

}
```

> <!-- TODO: Which version? -->
> ++ 2.2.0

Reviewer: I'm thinking 2.2.1 is fine for this? It's not a breaking change and not significant enough to push out to 2.3.0.

```suggestion
+++ 2.2.1
```

}
```

+++ 2.2.1

>     In case the `/metrics` endpoint is unacceptably slow to respond, you can disable specific apps that may be causing problems with the `METRICS_DISABLED_APPS` configuration setting.

Reviewer 2: ```suggestion
    In case the `/metrics` endpoint is not performant or not required slow, you can disable specific apps with the `METRICS_DISABLED_APPS` configuration setting.
```
